### PR TITLE
fix: new transferable balances calculation

### LINKED ISSUE #1388: Wrong transferable balance
The way transferable balance might not be updated to the current `balance` pallet.
Example:
Account 15aVbh9j99DyEJo9pYDs6UMB25ybHAteb4xmjnXkiBx855Qo shows a transferable balance of 0 in Talisman:
image

While a free balance of 8.47 DOT (at the time of writing) querying the chain state (`system.account`):
image

Related:
https://github.com/polkadot-js/apps/issues/9988
https://github.com/paritytech/polkadot-sdk/issues/1833#issuecomment-1805764506

### PR DESCRIPTION
**Pull request by alecdwm**

### CODE REVIEW

File: packages/balances-react/package.json [hunk 1]
```diff
@@ -52,12 +50,16 @@
...
     "jest": "^29.7.0",
+    "jotai": "^2.6.4",
+    "jotai-effect": "^0.5.0",
     "react": "^18.2.0",
     "ts-jest": "^29.1.1",
     "typescript": "^5.2.2"
   },
   "peerDependencies": {
     "@polkadot/util-crypto": "12.x",
+    "jotai": "*",
```
Comment (chidg) [inline packages/balances-react/package.json]: Not sure if this is good, we should not expect that all consuming apps will have or want to have jotai

**Review by chidg — COMMENTED** [review]
LGTM - just 1 question about peer deps

File: packages/balances-react/package.json (hunk 1 above)
Comment (0xKheops) [reply packages/balances-react/package.json]: concur

**Review by 0xKheops — COMMENTED** [review]
LGTM with same concern as chid

**Comment by alecdwm** [comment]
Sweet! I'll make another PR for the `@talismn/balances-atoms` package

**Review by 0xKheops — APPROVED** [review]
LGTM

**Pull request merged**